src/user/users-storage.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@nestjs/common';
+import { UpdateUserDto } from './dto/update-user.dto';
+import { User } from './entities/user.entity';
+
+@Injectable()
+export class UsersDataBase {
```

Comment:
I think we should move this logic to UserSrevice or make full service from this file